Orange/canvas/application/canvasmain.py

```diff
@@ -539,13 +505,14 @@ def setup_actions(self):
                     )
 
         self.show_output_action = \
-            QAction(self.tr("Show Output View"), self,
-                    toolTip=self.tr("Show application output."),
-                    triggered=self.show_output_view,
+            QAction(self.tr("Show Output Dock"), self,
```

Comment:
Could this be renamed to Log?

And should we redirect all logs that currently go to the console to this window (i.e. when run with -l 4 flag)?